addon/ng2/blueprints/component/files/__path__/__name__.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component} from 'angular2/core';
+
+@Component({
+  moduleId: __moduleName,
+  selector: '<%= dasherizedModuleName %>',
+  templateUrl: '<%= dasherizedModuleName %>.component.html',
+  styleUrls: ['<%= dasherizedModuleName %>.component.<%= styleExt %>']
+})
+export class <%= classifiedModuleName %>Component {
```

Comment:
I also prefer it being there. And the interface always goes with it, IMO. Otherwise folks are apt to change a spelling of it and then it never runs. Uh oh

I vote with Mike and say keep it for now.